### ISSUE TITLE
t/openapi-set-request.t fails

### ISSUE THREAD

**Issue by eserte** [jhthorsen/json-validator#23]
See subject. The test seems only to not fail if Mojolicious::Plugin::OpenAPI is not installed. Otherwise I see failures like:
```
#   Failed test '200 OK'
#   at t/openapi-set-request.t line 27.
#          got: '500'
#     expected: '200'

#   Failed test 'exact match for JSON Pointer "/bool"'
#   at t/openapi-set-request.t line 27.
#     Structures begin differing at:
#          $got = undef
#     $expected = 0

#   Failed test 'exact match for JSON Pointer "/bool"'
#   at t/openapi-set-request.t line 28.
#     Structures begin differing at:
#          $got = undef
#     $expected = 1

#   Failed test 'exact match for JSON Pointer "/bool"'
#   at t/openapi-set-request.t line 29.
#     Structures begin differing at:
#          $got = undef
#     $expected = 1

#   Failed test 'exact match for JSON Pointer "/this_stack/whatever"'
#   at t/openapi-set-request.t line 31.
#          got: undef
#     expected: 'something'

#   Failed test 'exact match for JSON Pointer "/whatever"'
#   at t/openapi-set-request.t line 31.
#          got: undef
#     expected: 'something'
# Looks like you failed 6 tests of 13.
t/openapi-set-request.t ...............
Dubious, test returned 6 (wstat 1536, 0x600)
Failed 6/13 subtests
```

**Comment by eserte** [comment]
My analysis is wrong. There may be pass reports if Mojolicious::Plugin::OpenAPI is installed.

**Comment by eserte** [comment]
Or maybe it's like this: the test fails if an older version of Mojolicious::Plugin::OpenAPI (e.g. 0.02) is installed.

Unfortunately upgrading is nearly impossible in this situation: Mojolicious::Plugin::OpenAPI requires the newest JSON::Validator, but this fails if an old Mojolicious::Plugin::OpenAPI is installed... only uninstalling (or hiding) Mojolicious::Plugin::OpenAPI helps.

**Comment by jhthorsen** [comment]
Do you think this would help?

```
plan skip_all => 'Mojolicious::Plugin::OpenAPI is required'
  unless eval "use Mojolicious::Plugin::OpenAPI 0.07";
```

**Comment by jhthorsen** [comment]
Closed by 0.78.
